### ISSUE TITLE
B.1.617.2 sublineage with K417N

### ISSUE THREAD

**Issue by rambaut** [cov-lineages/pango-designation#95]
There is a sub-clade of lineage B.1.617.2 that has `spike K417N` - considered to be an RBD mutation of concern. 

<img width="780" alt="image" src="https://user-images.githubusercontent.com/686405/119803449-5f3cd380-bed7-11eb-8993-62f4832e763c.png">

This is detected in multiple countries with quite a few cases list as having travel history suggesting it has arisen somewhere else and is at high enough incidence to be exported regularly. 

I propose this may be a sublineage of B.1.617.2 due to the very high concern about `spike K417N`. 

However, it only has K417N as a distinguishing mutation (and there may be another sub-lineage with this mutation within B.1.617.2). Most of the diversity seen has an additional mutation (`spike W258L`) to help assignment. 

[B.1.617.2_K417N.csv](https://github.com/cov-lineages/pango-designation/files/6553440/B.1.617.2_K417N.csv)

**Comment by AngieHinrichs** [comment]
FWIW due to an apparent back-mutation on S:G142D in some sequences, UShER places the sequences in two clusters (in red here):

https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-95.json?branchLabel=Spike%20mutations&c=userOrOld

The upper one has just B.1.617.2 + (16 mutations) + S:K417N, while the lower one has B.1.617.2 + (16 mutations) + back-mutation S:D142G + S:K417N.  S:G142D (G21987T) seems to blink on and off quite a bit in UCSC's tree.  

These related sequences are B.1.617.2 + (16 mutations) + the back-mutation S:D142G and _without_ K417N:

India/CG-AIIMS-Raipur-L-3566/2021|EPI_ISL_2098721|2021-04-02
France/NAQ-HCL021079463901/2021|EPI_ISL_1904998|2021-04-29
India/DEL-NCDC-INSACOG-2209272/2021|EPI_ISL_1970274|2021-04-07
Australia/WA680/2021|EPI_ISL_1669119|2021-04-16

-- so they anchor a node with that back-mutation.  Then the following sequences from your list have S:G142 (either back-mutation or lack of mutation in the first place or just noise?), plus K417N, so they get pulled down to that node to form the lower cluster:

USA/WA-CDC-UW21050386909/2021|EPI_ISL_2161928|2021-05-03:21986-21987
Switzerland/VD-HUG-34113638/2021|EPI_ISL_2213577|2021-05-08:21986-21987
Portugal/PT7214/2021|EPI_ISL_2248796|2021-05-01:21986-21987
Portugal/PT7250/2021|EPI_ISL_2248832|2021-05-07:21986-21987

And then Portugal/PT7244/2021|EPI_ISL_2248826|2021-05-11:21986-21987 has N at 21987, but is otherwise similar to Portugal/PT7214 so it gets pulled into that cluster.

The top cluster also has 3 sequences from Nepal:

Nepal/DH_011/2021|EPI_ISL_2171281|2021-04-30
Nepal/DH_007/2021|EPI_ISL_2171242|2021-05
Nepal/DH_008/2021|EPI_ISL_2171265|2021-05

What I'm calling B.1.617.2 is the branch with this sequence of mutations in UCSC's tree:
C14408T
C241T
A23403G
C3037T
G29742T
T22917G
T27638C
G28881T
G210T
C21618G
G29402T
C23604G
G23012C
G24410A
C25469T
G21987A
C22995A,C23012G,A28461G  (S:T478K, back-mutation S:Q484, N:D63G)

And the 16 subsequent mutations on the way to your cluster:
```
C16466T,C27752T                ORF1ab:C16202T, ORF7a:T120I
G15451A,T26767C                ORF1ab:G15187A, M:I82T
C10029T,C19220T,G28916T        ORF1ab:T3255I, ORF1ab:A6319V, N:G215C
G4181T                         ORF1ab:A1306S
C7124T,C8986T,G9053T,C27874T   ORF1ab:P2287S, ORF1ab:D2907D, ORF1ab:V2930L, ORF7b:T40I
A11332G                        syn
A11201G                        ORF1ab:T3646A
C6402T                         ORF1ab:P2046L
C21846T                        S:T95I (common ancestor of the 2 clusters with your sequences)
```

**Comment by rambaut** [comment]
Two of the defining mutation sites in B.1.617.2 (142 and 950) are in primer binding sites and there seems to be a tendency for some labs to call as reference (presumably because of not trimming primers in the consensus building). Strongly recommend masking at least 142 for building B.1.617.2

The tree Nepalese ones are ambiguous at site 417 (as are two German sequences) - and 417 is also in a primer site so may be a similar problem.

**Comment by rambaut** [comment]
Can confirm - masking spike 142 and 950 (nuc 21987 & 24410) results in a clean single cluster of 417N genomes. There are a few others with ambiguities at 417N so they may be mixed samples.

**Comment by AngieHinrichs** [comment]
Yes, makes sense, thanks for the info about the primer binding sites.

**Comment by babarlelephant** [comment]
About G142D  (G21987A), I looked at B.1.617.2 aligned trimmed (masked and unmasked) UK sequences and there is a NNNNNNN hotstop containing, often in region 21880:21991.

I did the same with B.1.1.7 sequences and there is no such NNNNNNN hotspot!

clades.nextstrain.org doesn't seem to complain about mutations in Artic amplicons. Also I found that the nucleotide motif GGTGTTTATTACC at 21986 is repeated at 21665. @AngieHinrichs @rambaut
